gulpfile.js

```diff
@@ -1,15 +1,31 @@
 const gulp = require('gulp');
-const obt = require('origami-build-tools');
+const process = require('gulp-preprocess');
+const argv = require('yargs').argv;
 
-gulp.task('build', function() {
-	
-	return obt.build.js({
-		js: './src/scripts/main.js',
-		buildJs: 'main.js',
-		buildFolder: './build/scripts/',
-		sourcemaps : false
-	});
+gulp.task('process', function() {
+	
+	var host = 'http://ftlabs-screens.herokuapp.com';
 
+	if(argv.deploy === 'test'){
+		host = 'http://ftlabs-screens-test.herokuapp.com';	
+	}
+	
+	gulp.src(['./src/scripts/main.js'])
+		.pipe( process( { context : {
+				host : host
+			}
+		}) )
+		.pipe( gulp.dest('./process/scripts/') )
+	;
+	
+	gulp.src(['./src/manifest.json'])
+		.pipe( process( { context : {
+				test : "-TEST"
```

Comment:
It was meant to add -TEST to the manifest.json file to indicate that the deployment points at the test servers, rather than the live ones. I couldn't get it to work, so I removed the placeholders for manifest.json, but I'll remove this now.